lib/tasks/import.rake

```diff
@@ -21,18 +21,14 @@ namespace :import do
     organisation_count = 0
     child_organisation_count = 0
     relationships = {}
-    host = ENV.fetch('LEGACY_HOST')
+    host = 'https://data.gov.uk/'
```

Comment:
Note that dumps on test.data.gov.uk are not currently available. Max looking into this. Once fixed then will add an env var